docs/CONTRIBUTING.md

```diff
@@ -203,7 +209,7 @@ yarn mode:account
 
 Note that only budibase maintainers will be able to access the pro repo. 
 
-The `yarn bootstrap` command can be used to replace the NPM supplied dependency with the local source aware version. This is achieved using the `yarn link` command. To see specifically how dependencies are linked see [scripts/link-dependencies.sh](../scripts/link-dependencies.sh). The same link script is used to link dependencies to account-portal in local dev. 
+The `yarn clean && yarn bootstrap` command can be used to replace the NPM supplied dependency with the local source aware version. This is achieved using the `yarn link` command. To see specifically how dependencies are linked see [scripts/link-dependencies.sh](../scripts/link-dependencies.sh). The same link script is used to link dependencies to account-portal in local dev. 
```

Comment:
Same as above around not needing `clean` here. All it will do is wipe down the `node_modules` directory of each package prior to bootstrapping